.circleci/config.yml

```diff
@@ -61,378 +61,9 @@ commands:
             - node_modules
 
 jobs:
-  build-client-integration:
-    docker:
-      - image: $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/ef-cms-us-east-1:2.12.0
-        aws_auth:
-          aws_access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
While I love the cleanup, I think we will need these tests for the merged code on the `test` branch/environment. Can you reimplement those for that branch/env?